src/components/button.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ButtonHTMLAttributes, ReactElement } from "react"
+import ApplePodcastIcon from "./applePodacstIcon"
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children?: any
+  variant?: "outline" | "solid" | "ghost"
+  as?: string
+  size?: string
+  href?: string
+  url?: any
+  alt?: any
+  key?: any
+}
+
+export default function Button({
+  variant = "solid",
+  children,
+  size,
+  as,
+  href,
+  url,
+  alt,
+  key,
+  ...props
+}: ButtonProps) {
+  const classNamesDefault = "py-2 px-4  font-semibold rounded-lg "
+
+  let classNamesVariant = " text-white   font-merri m-4 text-xs "
+  let classNamesSize = ""
+  if (variant === "outline") {
+    classNamesVariant =
+      "hover:no-underline text-blue-500 font-merri border-blue-500 border-2 border-solid m-0 hover:border-blue-600 hover:text-blue-600"
+  }
+
+  if (variant === "solid") {
+    classNamesVariant =
+      "no-underline  bg-blue-500 text-white font-merri text-md m-0  hover:bg-blue-600"
+  }
+  if (variant === "ghost") {
+    classNamesVariant = "flex text-black-900 font-merri text-md  py-4 px-8"
+  }
+  if (size === "s") {
+    classNamesSize = "py-4 px-4"
+  }
+  if (size === "m") {
+    classNamesSize = "py-5 px-5"
+  }
+  if (size === "l") {
+    classNamesSize = "py-6 px-6"
+  }
+  if (as === "a") {
+    return (
+      <a
+        href={`${href}`}
```

Comment:
you don't really need to interpolate here
keep it simple with

```suggestion
        href={href}
```